tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -33,8 +33,34 @@ class XlaGpuDeviceFactory : public DeviceFactory {
  public:
   Status CreateDevices(const SessionOptions& options, const string& name_prefix,
                        std::vector<std::unique_ptr<Device>>* devices) override;
+  // Returns a set containing the device ids contained in visible_device_list or
```

Comment:
Can this be a standalone function rather than a class member?

Sorry I didn't see this earlier.

If you move it to a standalone function, please mark it as `static`.

Reply:
Done